examples/rustfmt/client/BUILD

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2019 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cc_binary(
+    name = "client",
+    srcs = ["rustfmt.cc"],
+    deps = [
+        "//examples/rustfmt/proto:rustfmt_cc_grpc",
+        "//examples/utils",
+        "//oak/client:application_client",
+        "//oak/client:manager_client",
+        "@com_github_grpc_grpc//:grpc++",
+        "@com_google_absl//absl/flags:flag",
+        "@com_google_absl//absl/flags:parse",
+        "@com_google_absl//absl/memory",
+        "@com_google_asylo//asylo/util:logging",
```

Comment:
:( I know we don't have anything different, but having asylo as a dependency for client code is not great. I'll try to bring glog in today.